pages/projects_page.py

```diff
@@ -1,30 +1,116 @@
 import allure
+from selenium.webdriver.common.by import By
 
 from pages.top_bars.top_navigate_bar import TopNavigateBar
+from selenium.webdriver.support import expected_conditions
 
 
 class ProjectsPage(TopNavigateBar):
     """ Projects page - Where projects are added and edited"""
 
+    _START_BTN = (By.CSS_SELECTOR, "#app .px-4 a")
+    _CREATE_NEW_WORKSPACE_BTN = (By.CSS_SELECTOR, ".font-medium button")
+    _WORKSPACE_EDIT_BTN = (By.CSS_SELECTOR, "[data-icon='chevron-down']")
```

Comment:
change BTN to button in the entire project

Reply:
BUTTON you mean?